glib-macros/tests/value_delegate_derive.rs

```diff
@@ -31,10 +31,10 @@ fn into_value() {
         }
     }
 
-    test_func(&Test(123));
+    test_func(Test(123));
     test_func(Test(123));
```

Comment:
This is defeating the point of the test :D It should check that both values and references work